packages/cpp/README.md

```diff
@@ -102,6 +102,58 @@ There are two ways to configure clang-tidy's checks: through a Theia preference
     - for the .clang-tidy file: Checks: "-*,readability-*"
         - Meaning: disable all list-checks and enable all readability-* checks
 
+### Running clang-tidy as a task
+
+To be able to run clang-tidy as a task, you need to install the program.
+
+```bash
+UNIX
+    sudo apt-get install clang-tidy
+Note: not all operating system are currently supported with this linter yet.
+```
+
+```bash
```

Comment:
Why `bash`?
It renders the content oddly with certain words colored (read, source, do, for)

Reply:
Removed, Done :)